shopinvader_quotation/tests/test_quotation_download.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.shopinvader.tests.common import CommonCase, CommonTestDownload
+
+
+class QuotationDownloadCase(CommonCase, CommonTestDownload):
+    def setUp(self, *args, **kwargs):
+        super(QuotationDownloadCase, self).setUp(*args, **kwargs)
+        self.quotation = self.env.ref("shopinvader.sale_order_2")
```

Comment:
move everything but service init to setUpClass

Reply:
@simahawk done